Products/Jobber/model.py

```diff
@@ -380,6 +380,37 @@ def _save_record(log, storage, record):
         return False
 
 
+@inject_logger(log=_mlog)
+def save_scheduled_jobrecord(log, task_id=None, task=None, *args, **kwargs):
+    """Save the Zenoss scheduled job metadata to redis.
+    This function is registered as a handler for the task_prerun
+    signal.
+    """
+    task = get_app().tasks.get(task.name)
+    if task is None:
+        log.warn("Ignoring unknown task: %s", task.name)
+        return
+
+    if task.ignore_result:
+        log.debug("skipping; task result is ignored")
+        return
+
+    storage = getUtility(IJobStore, "redis")
+
+    record = RedisRecord.from_task(task, task_id,  args, kwargs)
+    record.update(
+        {
+            "status": states.PENDING,
+            "created": time.time(),
+        }
+    )
+
+    saved = _save_record(log, storage, record)
```

Comment:
I realize that `_save_record` already has a check for whether the job record already exists, but I think there should be a check here before calling `_save_record` to avoid seeing the "No user ID submitted with job ..." error twice in the log.

Reply:
I added a check before _save_record. 
We agree that there are no other suitable signal 'candidates' to use instead of task_prerun